src/plugins/LADTable/LADTableSetViewProvider.js

```diff
@@ -38,7 +38,7 @@ define([
             canEdit: function (domainObject) {
                 return domainObject.type === 'LadTableSet';
             },
-            view: function (domainObject) {
+            view: function (domainObject, isEditing, options) {
```

Comment:
Let's just pass the `objectPath` straight in. Maybe it's a minor point of nomenclature, but while the `objectPath` is "optional" in the sense the views can ignore it, it's not an "option" that determines the view's rendering or behavior.

Reply:
Also, we need to update the JSDoc in the `ViewRegistry` to document this new option.